src/components/invite/useInvites.js

```diff
@@ -180,13 +184,15 @@ const useInvited = () => {
     }
   }
 
+  const refresh = () => Promise.all([updateData(), updateInvited()])
+
   useEffect(() => {
-    updateInvited()
+    refresh().then(() => setInitialized(true))
```

Comment:
i need 'ininitalized' (e.g. loaded) flag - setting it after **both** data/invitees loaded